app/api/helpers/system_mails.py

```diff
@@ -188,7 +188,8 @@
         'message': (
             u"Hi, this is a confirmation mail of your tickets for the event {event_name}"
             u"<br/>Your order has been processed successfully."
-            + u"<br/> <a href='{pdf_url}'>Click here</a> to view/download your ticket."
```

Comment:
Can we also remove the pdf_url attribute from the api/mail.py in `send mail attendee function`?

Reply:
I'm not sure I understand. Could you point me to the permalink?

Reply:
```py
                html=MAILS[TICKET_PURCHASED]['message'].format(
                    pdf_url=holder.pdf_url,
                    event_name=order.event.name,
                    frontend_url=get_settings()['frontend_url'],
                ),
```
pdf_url here can be removed after this change .